### ISSUE TITLE
Project re-validation should be triggered when changing one of the configuration elements of a linter

### ISSUE THREAD

**Issue by orlandoibm** [angelozerr/tern.java#393]
Take this example:
1. Have a simple project
2. Convert it to Tern Project
3. Enable JsHint validation (at this point, tern.java offers to make a validation since Validation Settings have changed, which is good)
4. Open preferences and make a change in one of the JsHint flags (let's say bitwise), then apply the changes.
5. At this point, Validation Settings have changed, but since no linter has been added or removed from tern project, the onLintersChanged event is not fired, then the re-validation is not offered as it should.

**Comment by angelozerr** [comment]
@orlandoibm I agree with you. It should be very cool to do that when you update and save the .jshintrc.

**Comment by orlandoibm** [comment]
@angelozerr I actually was thinking about the case when you use a "config object" directly in .tern-project instead .jshintrc file, but certainly supporting both cases would be great.

**Comment by angelozerr** [comment]
@orlandoibm I have fixed the problem. For your information, you can update the .tern-project from the editor. See the following demo:

![test2](https://cloud.githubusercontent.com/assets/1932211/12655709/8504c3b0-c5fa-11e5-859b-3ce79bfd8ad0.gif)

**Comment by jabby** [comment]
@angelozerr Nice !

**Comment by angelozerr** [comment]
@jabby glaid it please you:)

Next step is to do the sam ething when .jshintrc is updated. I thinj it's a cool feature when you want to check quickly your jshint config.
